client/ReviewModule.jsx

```diff
@@ -69,8 +103,12 @@ export default class ReviewModule extends React.Component {
           name={name}
           handleTextChange={this.handleTextChange}
           searchText={searchText}
+          handleHover={this.handleStarHover}
```

Comment:
i think Airbnb wants to set off operators with spaces. For example, handleHover = {this.handleStarHover}